org.opent2t.sample.lamp.superpopular/com.contosothings.lightbulb/js/thingTranslator.js

```diff
@@ -23,11 +23,17 @@ const lightDeviceDi = 'F8CFB903-58BB-4753-97E0-72BD7DBC7933';
 // This translator class implements the 'org.opent2t.sample.lamp.superpopular' interface.
 class Translator {
 
-    constructor(deviceInfo) {
+    constructor(logger, deviceInfo) {
+        this.name = "opent2t-translator-com-contosothings-lightbulb";
+        this.logger = logger;
+        this.logger.verbose('Wink Lamp initializing...');
```

Comment:
I don't believe the first half of this log statement is useful, there is nothing big happening in between, thoughts?